numpydoc/numpydoc.py

```diff
@@ -132,6 +132,7 @@ def mangle_docstrings(app, what, name, obj, options, lines):
            app.config.numpydoc_show_inherited_class_members,
            'class_members_toctree': app.config.numpydoc_class_members_toctree}
 
+    cfg.update(options or {})
```

Comment:
Is this an appropriate thing to do with already-supported options?

Reply:
At some point numpydoc should re-use the sphinx versions of the options (`numpdoc_show_inherited_class_members` => 'inherited_members') rather than duplicate them, so I would claim it is  not only appropriate but desired